dogstatsd/alpine/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM alpine:3.4
+
+MAINTAINER Datadog <package@datadoghq.com>
+
+ENV DD_HOME=/opt/datadog-agent \
+    DOCKER_DD_AGENT=yes \
+    AGENT_VERSION=5.8.5 \
+    # prevent the agent from being started after install
+    DD_START_AGENT=0
+
+# Add install and config files
+ADD https://raw.githubusercontent.com/DataDog/dd-agent/$AGENT_VERSION/packaging/datadog-agent/source/setup_agent.sh /tmp/setup_agent.sh
+COPY entrypoint.sh supervisor.conf /
+
+# Expose supervisor and DogStatsD port
+EXPOSE 9001/tcp 8125/udp
+
+# Install minimal dependencies
+RUN apk add -qU --no-cache curl-dev python-dev tar sysstat
```

Comment:
why is `python-dev` necessary for run time? isn't it just required to install dependencies?